shuup/front/apps/carousel/admin_module/forms.py

```diff
@@ -38,17 +39,18 @@ class Meta:
         exclude = ("carousel",)
         widgets = {
             "active_dot_color": HexColorWidget(),
-            "inactive_dot_color": HexColorWidget()
+            "inactive_dot_color": HexColorWidget(),
+            "category_link": QuickAddCategorySelect(editable_model="shuup.Category"),
         }
 
     def __init__(self, **kwargs):
         self.carousel = kwargs.pop("carousel")
         self.request = kwargs.pop("request")
         super(SlideForm, self).__init__(**kwargs)
-
+        category_qs = Category.objects.all_except_deleted(shop=self.request.shop).prefetch_related('translations')
         self.empty_permitted = False
         shop = get_shop(self.request)
-        self.fields["category_link"].queryset = Category.objects.filter(shops=shop)
+        self.fields["category_link"].queryset = category_qs
```

Comment:
Put this same line. `self.fields["category_link"].queryset = Category.objects.all_except_deleted(shop=self.request.shop).prefetch_related('translations')`.

Reply:
Fixed